src/components/Checkout/DonationItem.vue

```diff
@@ -104,7 +104,7 @@
 			>
 				<button
 					class="donation-amount"
-					data-testid="basket-donation-edit-button"
+					data-testid="basket-donation-edit-button-combined"
```

Comment:
So long as you are testing on a viewport >= 734px wide this one will be visible/active. Below that number and the two above would be visible/active.